### PR TITLE
Creating strands_bringup 

### PR DESCRIPTION
**Pull request by Jailander**
removing strands_bob, strands_linda, strands_rosie, strands_uol_sim and strands_werner

### CODE REVIEW

**Comment by marc-hanheide** [comment]
this clearly didn't work:

```
CMake Error: Error in cmake code at
/var/lib/jenkins/workspace/pr-hydro-strands_systems/monitored_vcs/strands_bringup/CMakeLists.txt:16:
Parse error.  Function missing ending ")".  End of file reached.
```

did it work in your catkin workspace???

File: strands_bringup/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES scitos_bringup
+#  CATKIN_DEPENDS other_catkin_pkg
+#  DEPENDS system_lib
+)
+
+
+install(DIRECTORY launch
+   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
```
Comment (marc-hanheide) [inline strands_bringup/CMakeLists.txt:15]: obviously, there is at least a `)` missing here.

Comment (Jailander) [inline strands_bringup/CMakeLists.txt:15]: yes somehow I manage to delete the last line whilst removing other comments will fix now

File: strands_bringup/launch/strands_robot.launch [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
-    <machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>
-
-    <!-- SCITOS G5 Robot -->
+	<arg name="machine" default="$(optenv HOSTNAME localhost)" />
+	<arg name="user" default="" />
+
+	<arg name="head_camera" default="false" />
+	<arg name="head_ip" default="localhost" />
+	<arg name="head_user" default="strands" />
+
```
Comment (marc-hanheide) [inline strands_bringup/launch/strands_robot.launch:8]: You sure you want `<arg name="head_user" default="strands" />` ?

**Comment by marc-hanheide** [comment]
I'll wait to merge this until the above has been adressed, ok?

**Comment by Jailander** [comment]
yes sorry I missed that, now even the README is improved, sorry for all the commits

**Pull request merged**